tips/nomargin/main-no-margin.tex

```diff
@@ -1,22 +1,13 @@
 % !TEX program = LuaLaTeX+se
 
 % This document produces a pdf containing the score with no margin.
-% It is based on this solution:
-% http://tex.stackexchange.com/questions/291630/best-way-to-know-bottom-line-height-of-a-pdf/291664#291664
 %
-% It contains advanced commands quite difficult to understand, but should be straightforward
-% to adapt to your needs.
-%
-\documentclass[11pt]{article}
+\documentclass[11pt, varwidth=10cm]{standalone} % font size and score width
```

Comment:
Can you take this end of line comment, make it more verbose, and put it above this line?  That should make it easier for the novice to make use of the example.